charts/argo-rollouts/values.yaml

```diff
@@ -456,7 +456,8 @@ notifications:
     create: true
 
   secret:
-    # -- Whether to create notifications secret
+    # -- Whether to create notifications secret.
+    ## If you want to manually create secret, do not forget to add proper label to it: "app.kubernetes.io/component: rollouts-controller".
```

Comment:
Thank you for your PR.
nits; The label in secret is `app.kubernetes.io/component: {{ .Values.controller.component }}`, so it's not always true that `rollouts-controller` is the right value. 🤔 
Ref: https://github.com/argoproj/argo-helm/blob/argo-rollouts-2.37.6/charts/argo-rollouts/templates/controller/notifications-secret.yaml#L14

Reply:
Hi @st-myk 
thank you your update. Please run `./scripts/helm-docs.sh` to pass CI, as following below, the PR description.
> I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)